### PR TITLE
Integrated Circuits Parameter Fixes

### PR DESCRIPTION
**Pull request by Garen7**
[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl: Garen
fix: Added missing parameter types to circuits.
fix: Fixes two datatypes appearing for some input circuits.
tweak: Logic circuits now output booleans rather than 1's and 0's, this is purely cosmetic.
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)
Some circuits were missing parameter types allowing players to input values that would no matter what fail. Some of those also had the parameter type wrote into the name to attempt to make it look like the typing was correct however this just made two of the texts saying what type it is appear. Logic gates now output booleans rather than 1's and 0's(purely cosmetic) and had their descriptions updated accordingly.

### CODE REVIEW

File: config/admins.txt [hunk 1]
```diff
@@ -131,4 +131,5 @@ JStheguy = Game Master
 excessiveuseofcobby = Game Master
 Plizzard = Game Master
 octareenroon91 = Game Master
-Serpentarium = Game Master
+Serpentarium = Game Master
+Averagejoe82 = Game Master
```
Comment (ShizCalev) [inline config/admins.txt]: reeeeeeee

Comment (AverageJoe82) [reply config/admins.txt]: Is this a bad thing? I thought the admin.txt on the GitHub didn’t matter

Comment (PKPenguin321) [reply config/admins.txt]: Ye its fine

Comment (vuonojenmustaturska) [reply config/admins.txt]: yeah it doesn't matter

Comment (AverageJoe82) [reply config/admins.txt]: Danke

**Pull request merged**